jobserver/models/core.py

```diff
@@ -1049,10 +1053,15 @@ class BadLoginToken(Exception):
     class ExpiredLoginToken(Exception):
         pass
 
+    def _strip_token(self, token):
+        return token.strip().replace(" ", "")
+
     def generate_login_token(self):
         """Generate, set and return single use login token and expiry"""
         token = human_memorable_token()
-        self.login_token = make_password(token, salt="login-with-token")
+        self.login_token = make_password(
+            self._strip_token(token), salt="login-with-token"
```

Comment:
Would it be better to not provide a salt here to add more protection against rainbow table attacks? As I understand, django generates a random salt if one is not provided.